grammars/SeedPython.g4

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 The Aha001 Team.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+grammar SeedPython;
+
+/*
+ * parser rules
+ */
+
+prog: (NEWLINE | stmt)*;
+
+stmt: simple_stmt;
+
+simple_stmt: small_stmt NEWLINE;
+small_stmt: assign_stmt;
+assign_stmt: IDENTIFIER '=' expr;
+
+expr:
+	expr op = (MUL | DIV) expr		# mul_div
```

Comment:
Please replace all TABs with a corresponding number of spaces. No TAB is allowed in most Google code style guidelines.